Tasks/Common/VstsAzureRestHelpers_/VstsAzureRestHelpers_.psm1

```diff
@@ -1,8 +1,8 @@
 ﻿# Private module-scope variables.
 $script:jsonContentType = "application/json;charset=utf-8"
 $script:formContentType = "application/x-www-form-urlencoded;charset=utf-8"
-$script:azureRmUri = "https://management.azure.com"
-$script:authUri = "https://login.microsoftonline.com/"
+$script:defaultAuthUri = "https://login.microsoftonline.com/"
```

Comment:
login.windows.net is legacy auth URL, login.microsoftonline.com is latest auth url. In environment property its showing only latest one. Since other URL is being used at many places in application so added it as another property.